test/uk/gov/hmrc/tai/connectors/HttpHandlerSpec.scala

```diff
@@ -16,27 +16,40 @@
 
 package uk.gov.hmrc.tai.connectors
 
+import com.github.tomakehurst.wiremock.client.WireMock._
 import org.joda.time.LocalDate
-import org.mockito.Matchers._
-import org.mockito.Mockito._
-import org.mockito.{Matchers, Mockito}
-import org.scalatest.BeforeAndAfterEach
-import play.api.http.Status._
-import play.api.libs.json.{Format, JsString, Json}
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.{MustMatchers, WordSpec}
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.http.Status.{INTERNAL_SERVER_ERROR, _}
+import play.api.i18n.Messages
+import play.api.libs.json.{Format, Json}
+import play.api.test.Injecting
+import uk.gov.hmrc.domain.Generator
 import uk.gov.hmrc.http._
-import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
-import utils.BaseSpec
+import utils.WireMockHelper
 
+import scala.concurrent.Await
 import scala.concurrent.duration._
-import scala.concurrent.{Await, Future}
 
-class HttpHandlerSpec extends BaseSpec with BeforeAndAfterEach {
+class HttpHandlerSpec
+    extends WordSpec with GuiceOneAppPerSuite with MustMatchers with WireMockHelper with ScalaFutures
+    with IntegrationPatience with Injecting {
 
-  override def beforeEach: Unit =
-    Mockito.reset(http)
+  val generatedNino = new Generator().nextNino
+
+  val generatedSaUtr = new Generator().nextAtedUtr
```

Comment:
Do we need this? Not sure how a SaUtr is linked to an AtedUtr here? TAI also does not use SAUtrs anywhere.

Reply:
Removed them , done